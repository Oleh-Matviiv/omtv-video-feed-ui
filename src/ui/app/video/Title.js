import React from 'react';
import styled from 'styled-components';

const Title = styled.div.attrs({
  className: 'video-title',
})`
  padding: 1rem;
  grid-area: title;
`;

export default Title;
