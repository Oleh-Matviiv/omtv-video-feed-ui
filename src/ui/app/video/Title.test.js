import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';

import Title from './Title';

describe('snapshots', () => {
  it('Title', () => {
    const wrapper = mount(
      <Title>title</Title>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
