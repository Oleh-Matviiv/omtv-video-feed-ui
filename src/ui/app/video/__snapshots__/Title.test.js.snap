// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots Title 1`] = `
.c0 {
  padding: 1rem;
  grid-area: title;
}

<styled.div>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [
          Object {
            "className": "video-title",
          },
        ],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "
  padding: 1rem;
  grid-area: title;
",
          ],
        },
        "displayName": "styled.div",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "div",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <div
      className="video-title c0"
    >
      title
    </div>
  </StyledComponent>
</styled.div>
`;
